apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V96__recreate_pre_adj_child_tables.sql

```diff
@@ -0,0 +1,198 @@
+-- Rebuild the schema to match new RDA specifications and truncate tables to end soft launch
```

Comment:
lets make sure to ask the BFD team for a clone of the prod database and test this out

Reply:
As discussed elsewhere, I created a limited `paca-717` environment fashioned after the `prod` environment. From the [migrator logs](https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#logsV2:log-groups/log-group/$252Fbfd$252Fpaca-717$252Fbfd-db-migrator$252Fmigrator-log.json/log-events/i-06d4acef3f2ee82c5), _this v96 migration_<sup>1</sup> completed successfully in 1.297s:

```json
{
  "timestamp": "2022-09-02T11:43:47.314+0000",
  "level": "INFO",
  "thread": "main",
  "logger": "org.flywaydb.core.internal.command.DbMigrate",
  "message": "Successfully applied 1 migration to schema \"public\", now at version v96 (execution time 00:01.297s)",
  "context": "default"
}
```

<details><summary><strong>1</strong></summary>

> I'm just hedging on the version number. At the time of this writing, obviously the version was v96, but because we are doing linear and strictly increasing versions for our flyway migrations, this _could_ change if any other migrations precede it.

</details>

I'll be sure to work with @keithdadkins for any required maintenance activity, but as it stands now, I think we can accommodate this when the time comes.